tests/installation/sles4sap_wizard_nw.pm

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/perl -w
+use strict;
+use base "y2logsstep";
+use testapi;
+
+sub run() {
+    assert_screen "sles4sap-wizard-nw-swpm-welcome", 120;
+    send_key $cmd{"next"}, 1;
+    assert_screen "sles4sap-wizard-nw-swpm-params", 60;
+    type_string 'QNW',                              1;    # Sid
```

Comment:
Sid? I don't get the meaning of that comment - but type_string ,1 means it will type like a cat